docs/api/configuration.md

```diff
@@ -0,0 +1,81 @@
+
+## Example
+
+```
+{
+    "data_origin": "https://ncsa.osn.xsede.org/Pangeo/pangeo-forge/noaa-coastwatch-geopolar-sst-feedstock/noaa-coastwatch-geopolar-sst.zarr",
+    "data_array": {
+        "active": "analysed_sst",
+        "x": "lon",
+        "y": "lat",
+        "t": "time",
+        "t_index": 5
+    },
+    "data_slices": {
+        "lat": [
+            -45,
+            45,
+            100
+        ],
+        "lon": [
+            -90,
+            90,
+            100
+        ]
+    },
+    "ui": {
+        "main_drawer": false,
+        "axis_drawer": false,
+        "expanded_coordinates": []
+    }
+}
+```
+
+For more example configuration files, visit our [Examples on Github](https://github.com/Kitware/pan3d/tree/main/examples).
+
+
+## `data_origin` (Required)
+The value for this key should be a string containing a local path or remote URL, referencing a target dataset readable by `xarray.open_dataset`. This value will be passed to `DatasetBuilder.set_dataset_path`.
+
+## `data_array` (Required)
+The value for this key should be a mapping specifying how to interpret the information in the target dataset. The following table describes keys available in this mapping schema.
+
+| Key | Required? | Type | Value Description |
+|-----|-----------|------|-------------------|
+|`active`|YES     |`str` |This should be a name of an array that exists in the current dataset. This value will be passed to `DatasetBuilder.set_data_array_active_name`. |
+|`x`  |NO (default=None)  |`str`|This should be the name of a coordinate that exists in the active data array. This value will be passed to `DatasetBuilder.set_data_array_axis_names`.|
+|`y`  |NO (default=None)  |`str`|This should be the name of a coordinate that exists in the active data array. This value will be passed to `DatasetBuilder.set_data_array_axis_names`.|
+|`z`  |NO (default=None)  |`str`|This should be the name of a coordinate that exists in the active data array. This value will be passed to `DatasetBuilder.set_data_array_axis_names`.|
+|`t`  |NO (default=None)  |`str`|This should be the name of a coordinate that exists in the active data array. This value will be passed to `DatasetBuilder.set_data_array_axis_names`.|
+|`t_index` |NO (default=0)|`int`|The index of the current time slice. Must be an integer >= 0 and < the length of the current time coordinate.This value will be passed to `DatasetBuilder.set_data_array_time_index`.|
+
+## `data_slices` (Optional)
+The value for this key should be a mapping of coordinate names (which are likely used as values for `x` | `y` | `z` | `t` in the `data_array` section) to slicing arrays. This mapping will be formatted and passed to `DatasetBuilder.set_data_array_coordinates`.
+
+Each slicing array should be a list of three values `[start, stop, step]`.
+
+`start`: the coordinate value at which the sliced data should start (inclusive)
+
+`stop`: the coordinate value at which the sliced data should stop (exclusive)
```

Comment:
We could add a note saying that we allow negative indexing?

Reply:
We currently don't allow for negative slice values, though we could make that change. As for the stop and start values, these should be values that exist in the coordinate space instead of indexes. For example, I can use [-90, 90, 5] for latitude. Is there a way I can make this more clear in this text?